### DISCUSSION TITLE
docker crash on cluster does not exist WHD02 device

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#10453] [Devices]
Hi There,
I'm quite new to MQTT and zigbee and am trying to get zigbee2mqtt up and running in an docker container. This is actually working quite well, but when trying to connect my WHD02 switch the docker container crashes. The device is created within the config file but whenever it is connecting zigbee2mqtt crashes with the same error message "cluster does not exist" see below (last one in the log below). Any ideas?
Thanks a lot
`Zigbee2MQTT:info  2021-12-30 18:16:20: MQTT publish: topic 'zigbee2mqtt/bridge/config', payload '{"commit":"c2b5229","coordinator":{"meta":{"maintrel":3,"majorrel":2,"minorrel":6,"product":0,"revision":20211115,"transportrev":2},"type":"zStack12"},"log_level":"info","network":{"channel":11,"extendedPanID":"0x00124b001cd4e6ff","panID":6754},"permit_join":true,"version":"1.22.1"}'

Zigbee2MQTT:info  2021-12-30 18:23:36: Device '0xa4c1382aad447664' joined

Zigbee2MQTT:info  2021-12-30 18:23:36: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xa4c1382aad447664","ieee_address":"0xa4c1382aad447664"},"type":"device_joined"}'

Zigbee2MQTT:info  2021-12-30 18:23:36: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0xa4c1382aad447664"},"type":"device_connected"}'

Zigbee2MQTT:info  2021-12-30 18:23:36: Starting interview of '0xa4c1382aad447664'

Zigbee2MQTT:info  2021-12-30 18:23:36: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xa4c1382aad447664","ieee_address":"0xa4c1382aad447664","status":"started"},"type":"device_interview"}'

Zigbee2MQTT:info  2021-12-30 18:23:36: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0xa4c1382aad447664"},"type":"pairing"}'

Zigbee2MQTT:info  2021-12-30 18:23:36: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xa4c1382aad447664","ieee_address":"0xa4c1382aad447664"},"type":"device_announce"}'

Zigbee2MQTT:info  2021-12-30 18:23:36: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0xa4c1382aad447664"},"type":"device_announced"}'

Zigbee2MQTT:info  2021-12-30 18:23:37: Successfully interviewed '0xa4c1382aad447664', device has successfully been paired

Zigbee2MQTT:info  2021-12-30 18:23:37: Device '0xa4c1382aad447664' is supported, identified as: TuYa 1 gang switch (TS0001)

Zigbee2MQTT:info  2021-12-30 18:23:37: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"1 gang switch","exposes":[{"features":[{"access":7,"description":"On/off state of the switch","name":"state","property":"state","type":"binary","value_off":"OFF","value_on":"ON","value_toggle":"TOGGLE"}],"type":"switch"},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"TS0001","options":[],"supports_ota":false,"vendor":"TuYa"},"friendly_name":"0xa4c1382aad447664","ieee_address":"0xa4c1382aad447664","status":"successful","supported":true},"type":"device_interview"}'

Zigbee2MQTT:info  2021-12-30 18:23:37: Configuring '0xa4c1382aad447664'

Zigbee2MQTT:info  2021-12-30 18:23:37: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"description":"1 gang switch","friendly_name":"0xa4c1382aad447664","model":"TS0001","supported":true,"vendor":"TuYa"},"type":"pairing"}'

Zigbee2MQTT:info  2021-12-30 18:23:37: MQTT publish: topic 'homeassistant/switch/0xa4c1382aad447664/switch/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0xa4c1382aad447664/set","device":{"identifiers":["zigbee2mqtt_0xa4c1382aad447664"],"manufacturer":"TuYa","model":"1 gang switch (TS0001)","name":"0xa4c1382aad447664"},"json_attributes_topic":"zigbee2mqtt/0xa4c1382aad447664","name":"0xa4c1382aad447664","payload_off":"OFF","payload_on":"ON","state_topic":"zigbee2mqtt/0xa4c1382aad447664","unique_id":"0xa4c1382aad447664_switch_zigbee2mqtt","value_template":"{{ value_json.state }}"}'

Zigbee2MQTT:info  2021-12-30 18:23:37: MQTT publish: topic 'homeassistant/sensor/0xa4c1382aad447664/linkquality/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0xa4c1382aad447664"],"manufacturer":"TuYa","model":"1 gang switch (TS0001)","name":"0xa4c1382aad447664"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0xa4c1382aad447664","name":"0xa4c1382aad447664 linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0xa4c1382aad447664","unique_id":"0xa4c1382aad447664_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'

Zigbee2MQTT:info  2021-12-30 18:23:38: Successfully configured '0xa4c1382aad447664'

Error: Cluster with name '57344' does not exist

    at Object.getCluster (/app/node_modules/zigbee-herdsman/src/zcl/utils.ts:66:19)

    at Endpoint.saveClusterAttributeKeyValue (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:242:35)

    at Controller.onZclOrRawData (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:644:26)

    at ZStackAdapter.<anonymous> (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:135:70)

    at ZStackAdapter.emit (events.js:400:28)

    at ZStackAdapter.onZnpRecieved (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:796:30)

    at Znp.emit (events.js:400:28)

    at Znp.onUnpiParsed (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:108:18)

    at Parser.emit (events.js:400:28)

    at Parser.parseNext (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/unpi/parser.ts:45:26)
`

**Comment by Koenkk** [comment]
Same issue as #10174 which has already been fixed in the dev branch. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by ghost** [reply]
Ty, ill give it a try.

**Reply by ghost** [reply]
Problem solved, thanks a lot for your support and quick response
